show/main.py

```diff
@@ -129,6 +173,148 @@ def run_command(command, display_cmd=False):
     if rc != 0:
         sys.exit(rc)
 
+
+def get_interface_mode():
+    mode = os.getenv('SONIC_CLI_IFACE_MODE')
+    if mode is None:
+        mode = "default"
+    return mode
+
+
+# Global Alias object
+_interface = InterfaceAliasConverter()
```

Comment:
```
# Global class instance for SONiC interface name to alias conversion
convert_interface = InterfaceAliasConverter()
```
Renamed as convert_interface so, if name_to_alias / alias_to_name functions are used along with this variable name it will be more meaningful..

``` 
               if get_interface_mode() == "alias":
                    alias = convert_interface.name_to_alias(m)
                    r.append(alias)
                else:
                    r.append(m)

```

Reply:
"convert_interface" sounds like a function, not an object. It's hard not to get too verbose, but I think it should be named something like "iface_alias_converter".

Reply:
Addressed as discussed in new commit..